### PR TITLE
MJ Popquiz Jan13

### PR DESCRIPTION
**Pull request by merrilj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,12 @@ swap(one, -1, 0)
 console.log(one) // [1]
 
 function swap(array, first_index, second_index) {
-  // -- YOUR ANSWER HERE --
+  var temp = array[first_index]
+  array[first_index] = array[second_index]
+  array[second_index] = temp
+  return array
```
Comment (carlosnsr) [inline README.md:37]: Perfect.

File: README.md [hunk 2]
```diff
@@ -56,7 +60,10 @@ console.log(evens) // [2, 8, 6, 4, 10]
 reverse(evens)
 console.log(evens) // [10, 4, 6, 8, 2]
 
+// Hey, the world has evolved into the 21st century :/
+
 function reverse(array) {
-  // -- YOUR ANSWER HERE --
+  return array.reverse()
```
Comment (carlosnsr) [inline README.md:66]: Please re-write using the above `swap()` function

**Comment by carlosnsr** [comment]
Please revise and re-submit

**Comment by merrilj** [comment]
Ok it should be done.

**Pull request closed without merging**